### ISSUE TITLE
lispy-forward treats atoms and lists differently

### ISSUE THREAD

**Issue by Wilfred** [abo-abo/lispy#459]
I use `]` extensively for stepping over expressions, but it treats atoms differently from lists. For example,

```
(foo
 |bar
 baz)
```

here lispy-forward moves point to the closing `)`. I was expecting point to be at the end of `bar`. However, in this case:

```
(foo
 |(bar)
 baz)
```

lispy-forward puts point on the end of the line.

Is this intentional behaviour? I see that lispy-forward is bound to `C-M-f` in `lispy-mode-map-paredit`, but `paredit-forward` has the behaviour I expected.

Is this functionality available as a different lispy command (I couldn't see any), or something you'd be willing to change or make customisable?

**Comment by notmgsk** [comment]
I don't think it exists in lispy because it exists as a standard emacs command: `forward-sexp`.
